src/include/uapi/ipc/topology.h

```diff
@@ -216,7 +202,9 @@ enum sof_ipc_process_type {
 	SOF_PROCESS_NONE = 0,		/**< None */
 	SOF_PROCESS_EQFIR,		/**< Intel FIR */
 	SOF_PROCESS_EQIIR,		/**< Intel IIR */
-	SOF_PROCESS_KEYWORD_DETECT,     /**< Keyword Detection */
+	SOF_PROCESS_KEYWORD_DETECT,	/**< Keyword Detection */
+	SOF_PROCESS_KPB,		/**< KeyPhrase Buffer Manager */
+	SOF_PROCESS_CHAN_SELECTOR,	/**< Channel Selector */
```

Comment:
Looks like we don't use this enum in FW at all. Therefore I would remove it.

Reply:
thanks for reviewing @mrajwa, good point, let me remove this enum type from the header file from both FW and driver.